tests/conftest.py

```diff
@@ -74,17 +75,30 @@ def pytest_collection_modifyitems(session, config, items):
 
     """
     for item in items[:]:
-        skip_marker = item.get_closest_marker("skipif_ocs_version")
-        if skip_marker:
-            skip_condition = skip_marker.args
-            log.info(skip_condition)
-            # skip_confition will be a tuple
-            # and condition will be first element in the tuple
-            if skipif_ocs_version(skip_condition[0]):
-                log.info(
-                    f'Test: {item} will be skipped due to {skip_condition}'
-                )
-                items.remove(item)
+        skip_marker = [
+            item.get_closest_marker("skipif_ocs_version"),
+            item.get_closest_marker("skipif_lso_deployment")
+        ]
+        for marker in skip_marker:
+            if marker:
+                if marker.name == 'skipif_ocs_version':
```

Comment:
I don't think you need to have these conditions. The handling should be similar for both 'skipif_ocs_version' and 'skipif_lso_deployment'

Reply:
skipif_ocs_version takes input(version) as args.
skipif_lso_deployment function will not take any args(version as input), it checks whether the deployment is LSO based or not and returns boolean based on it.

Please suggest if there is a better way to implement.